src/pages/components/NavBar.tsx

```diff
@@ -1,18 +1,90 @@
-import { Anchor } from "@mantine/core";
+import { SignedIn, UserButton } from "@clerk/clerk-react";
+import { useLocation } from "react-router-dom";
+import styled from "styled-components";
+
+const NavBarContainer = styled.div`
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  background-color: green;
+  align-items: center;
+  padding-top: 10px;
+  border-bottom: 1px solid;
+`;
+
+const AnchorContainer = styled.div`
+  display: flex;
+`;
+
+const ActiveAnchor = styled.a`
+  background-color: white;
+  color: black;
+  padding: 20px;
+  margin-left: 10px;
+  border-top-left-radius: 5px;
+  border-top-right-radius: 5px;
+  border-left: black solid 1px;
+  border-right: black solid 1px;
+  border-top: black solid 1px;
+  border-bottom-color: transparent;
+  margin-bottom: -1px;
+  text-decoration: none;
+`;
+
+const InactiveAnchor = styled.a`
+  background-color: green;
+  color: black;
+  padding: 20px;
+  margin-left: 10px;
+  border-bottom-color: transparent;
+  margin-bottom: -1px;
+  text-decoration: none;
+`;
+
+const ProfileContainer = styled.div``;
```

Comment:
Again, is this just for clear components?

Reply:
yes will be styled later on